### ISSUE TITLE
NCIT_C42065	Mature Ganglioneuroma

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/monarch-disease-ontology-RETIRED#314]
NCIT_C42065 | Mature Ganglioneuroma | SubClassOf | DOID_4817 | ganglioneuroma
-- | -- | -- | -- | --

![image](https://user-images.githubusercontent.com/6722114/29097234-9a60890a-7c4d-11e7-9713-e03ff5045572.png)

**Comment by nicolevasilevsky** [comment]
I think this is okay, as NCIT already asserts this relationship.

**Comment by cmungall** [comment]
I agree!

However, before we close this we should investigate why the global probability of this was low enough to be flagged: It's the connection to gangliocytomas, note the low probabilities of the individual linkages. 

Do you think there should be a connection between ganglioneuroma and gangliocytoma?

**Comment by cmungall** [comment]
here is how different ontologies classify this (no inter-ontology linkages shown):

![image](https://user-images.githubusercontent.com/50745/29104165-b6838d8c-7c78-11e7-8568-30c3b6b194e8.png)
![image](https://user-images.githubusercontent.com/50745/29104197-e872ff9e-7c78-11e7-9ce5-ff1a867f7dd1.png)


 * NCIT:C3049 ! Ganglioneuroma [DEF: "A benign neuroblastic tumor of the sympathetic nervous system that occurs in childhood. Ganglioneuroma typically arises from the sympathetic trunk in the mediastinum. Histologic features include spindle cell proliferation (resembling a neurofibroma) and the presence of large ganglion cells. Common presenting features include a palpable abdominal mass, hepatomegaly, and a thoracic mass detected on routine chest X-ray."]
 * NCIT:C6934 ! Gangliocytoma [DEF: "A well differentiated, slow growing neuroepithelial neoplasm composed of neoplastic, mature ganglion cells."]

TL;DR - we dispute the DO xrefs, and we should assert these to be sibs

**Comment by nicolevasilevsky** [comment]
This is related to #317
